### PR TITLE
Build on Circle CI with Riffraff upload

### PR DESCRIPTION
**Pull request by jamespamplin**
Builds on [Circle CI](https://circleci.com/gh/guardian/editorial-viewer) and uploads to Riffraff S3 buckets for deployment.

### CODE REVIEW

**Comment by hmgibson23** [comment]
:+1:

File: circle.yml [hunk 1]
```diff
@@ -1,4 +1,32 @@
...
+    version: oraclejdk8
+
+# Collect build artifacts
+general:
+  artifacts:
+    - target/riffraff
 
 dependencies:
   pre:
     - ./setup.sh
```
Comment (jamespamplin) [inline circle.yml]: @clloyd do you think we need `./setup.sh` on build anymore? Or can we replace this with just a `npm install` as we have a post-install hook to do jspm things?

Comment (clloyd) [inline circle.yml]: No, it's not doing anything that `npm i` isn't doing.

Comment (jamespamplin) [inline circle.yml]: Great, I'll update this.

Comment (sihil) [inline circle.yml]: How do you install / spin up mongo?

Comment (clloyd) [inline circle.yml]: Viewer doesn't use mongo, setup.sh is just installing npm and jspm modules.

**Comment by jamespamplin** [comment]
Got a build generated from Circle CI on CODE. Builds now have the namespace `editorial-tools:viewer` in RiffRaff.

**Pull request merged**